docs/docusaurus/docusaurus.config.js

```diff
@@ -149,7 +149,7 @@ module.exports = {
         {
           type: 'doc',
           label: 'GX OSS',
-          docId: 'oss/oss',
+          docId: 'core/introduction/introduction',
```

Comment:
@Rachel-Reverie fyi I tried with just `core/introduction` and it didn't seem to work